packages/server/__snapshots__/5_spec_isolation_spec.js

```diff
@@ -80,14 +80,14 @@ exports['e2e spec isolation fails'] = {
           "error": "fail1\n\nBecause this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `beforeEach hooks`",
           "timings": {
             "lifecycle": 100,
-            "before all": [
+            "before all (1)": [
```

Comment:
Will this impact the dashboard at all? I worry it might be expecting there only to be a single "before all" and/or "before each" in the timings and might not handle having them broken up and appended with a number.